### PR TITLE
Jt 17

### PR DESCRIPTION
**Pull request by saffchen**

### CODE REVIEW

File: stock/src/main/java/saffchen/mapper/SatelliteMapper.java [hunk 1]
```diff
@@ -14,4 +14,6 @@ public interface SatelliteMapper {
     SatelliteDto satelliteToSatelliteDto(SatelliteEntity satellite);
 
     List<SatelliteDto> toSatellitesDtoList(List<SatelliteEntity> satelliteEntities);
+
+    SatelliteEntity satelliteEntityToSatelliteDtoToSatelliteEntity(SatelliteDto satellite);
```
[the remark this answers is not in the document]
Comment (saffchen) [reply stock/src/main/java/saffchen/mapper/SatelliteMapper.java:18]: This method name fully describes its purpose. Thus there will be no confusion

[the remark this answers is not in the document]
Comment (saffchen) [reply stock/src/main/java/saffchen/mapper/SatelliteMapper.java]: Thanks. Fixed

File: stock/src/main/java/saffchen/service/SatelliteService.java [hunk 2]
```diff
@@ -36,7 +37,15 @@ public SatelliteEntity getSatelliteById(Long id) throws NoEntityException {
...
+    public Long getProductCountBySatelliteId(Long id) {
         return Long.valueOf(productRepository.productCountBySatelliteId(id).size());
     }
+
+    public SatelliteEntity saveNewSatellite(SatelliteEntity satellite) {
+        SatelliteEntity satelliteIsExist = satelliteRepository.findByName(satellite.getName())
+                .orElse(null);
+        if (satelliteIsExist == null) {
+            return satelliteRepository.save(satellite);
+        } else throw new SatelliteAlreadyExistException("This satellite already has been exist.");
```
Comment (fstk1337) [inline stock/src/main/java/saffchen/service/SatelliteService.java:49]: Can be written without else block.

**Review by fstk1337 — APPROVED** [review]
Seems good.

**Pull request merged**